core/app/models/spree/ability.rb

```diff
@@ -85,6 +86,11 @@ def apply_user_permissions(user)
       can :read, ::Spree::Taxonomy
       can :read, ::Spree::Variant
       can :read, ::Spree::Zone
+      can :manage, ::Spree::Wishlist, user_id: user.id
+      can :read, ::Spree::Wishlist, is_private: false
```

Comment:
shouldn't we swap the order of these 2? won't the 2nd one overwrite the 1st one? don't remember how cancacan prioritizes this...

Reply:
Hi @damianlegawiec 

I have just been working on this now as I didn't feel it was optional, but I'm not sure if what I have changed is better, take a look at this next commit and if its not a better solution I'll revert it...